lte/gateway/python/scripts/runtime_report.py

```diff
@@ -46,17 +46,24 @@ def merge_all_report(working_dir, list_xml_report_paths, output_path):
         xml_file_path = working_dir + "/" + xml_file_path
         test_result = ET.parse(xml_file_path)
         test_suites_data = test_result.getroot()
-        # for the integration tests pytest generates the XML files with a slightly different structure
```

Comment:
Was the old name wrong?

Reply:
I introduced this conditional when I was working on publishing integration tests results, but now that unit tests require the same conditional I think it is reasonable to rename it.